sdk/python/feast/type_map.py

```diff
@@ -462,51 +358,7 @@ def pa_to_feast_value_type(value: Union[pa.lib.ChunkedArray, str]) -> ValueType:
         "list<item: binary>": ValueType.BYTES_LIST,
         "list<item: bool>": ValueType.BOOL_LIST,
     }
-    return type_map[value_type]
-
-
-def pa_column_to_timestamp_proto_column(column: pa.lib.ChunkedArray) -> List[Timestamp]:
-    if not isinstance(column.type, TimestampType):
-        raise Exception("Only TimestampType columns are allowed")
-
-    proto_column = []
-    for val in column:
-        timestamp = Timestamp()
-        timestamp.FromMicroseconds(micros=int(val.as_py().timestamp() * 1_000_000))
-        proto_column.append(timestamp)
-    return proto_column
-
-
-def pa_column_to_proto_column(
```

Comment:
So this function is not in use anywhere?

Reply:
It was never used (check https://github.com/feast-dev/feast/search?q=pa_column_to_proto_column)